<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="SoftwareManagement.sm_apt_install_download_path" skipped="0" tests="1" time="84.75" timestamp="2022-06-15T11:29:45">
	<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED"/>
		<system-out>11:28:20 INFO  ==============================================================
11:28:20 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:28:20 INFO  Dir:   software_management_end_to_end/
11:28:20 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:28:20 INFO  ==============================================================
11:28:21 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:28:21 INFO  EnvironmentC8y Setup
11:28:21 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:28:36 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:28:37 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:28:38 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:28:39 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:28:41 ERROR No operations found, assuming it passed
11:28:41 INFO  Assert that (False == value) with value=False ... passed
11:28:42 INFO  Executed sudo, exit status 0
11:28:42 INFO  Executed sudo, exit status 0
11:28:43 INFO  Executed sudo, exit status 0
11:28:47 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:28:47 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
11:28:47 INFO  Delaying execution by 7 seconds
11:29:08 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:29:08 INFO  Response status: 201
11:29:08 INFO  Response to action: {
    &quot;creationTime&quot;: &quot;2022-06-15T11:29:08.449Z&quot;,
    &quot;deviceName&quot;: &quot;m32sd10a&quot;,
    &quot;deviceId&quot;: &quot;20220944&quot;,
    &quot;self&quot;: &quot;https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20232104&quot;,
    &quot;id&quot;: &quot;20232104&quot;,
    &quot;status&quot;: &quot;PENDING&quot;,
    &quot;description&quot;: &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;,
    &quot;c8y_SoftwareUpdate&quot;: [
        {
            &quot;name&quot;: &quot;rolldice&quot;,
            &quot;action&quot;: &quot;install&quot;,
            &quot;id&quot;: &quot;5152439&quot;,
            &quot;version&quot;: &quot;::apt&quot;,
            &quot;url&quot;: &quot;https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463&quot;
        }
    ]
}
11:29:08 INFO  Started operation: {'creationTime': '2022-06-15T11:29:08.449Z', 'deviceName': 'm32sd10a', 'deviceId': '20220944', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20232104', 'id': '20232104', 'status': 'PENDING', 'description': &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;, 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
11:29:08 WARN  , retrying in 0.5 seconds...
11:29:09 WARN  , retrying in 0.5 seconds...
11:29:09 WARN  , retrying in 0.5 seconds...
11:29:10 WARN  , retrying in 0.5 seconds...
11:29:10 WARN  , retrying in 0.5 seconds...
11:29:11 WARN  , retrying in 0.5 seconds...
11:29:11 WARN  , retrying in 0.5 seconds...
11:29:12 WARN  , retrying in 0.5 seconds...
11:29:12 WARN  , retrying in 0.5 seconds...
11:29:13 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py&quot;, line 1294, in __call__
    self.testObj.execute()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py&quot;, line 232, in fun
    return caller(func, *(extras + args), **kw)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 74, in retry_decorator
    logger)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 33, in __retry_internal
    return f()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 28, in assert_install_in_download_path
    assert &quot;rolldice&quot; in os.listdir(f&quot;{install_directory}&quot;)
AssertionError
11:29:13 WARN  AssertionError: ... blocked 
11:29:13 INFO  
11:29:13 INFO  cleanup:
11:29:14 INFO  Executed sudo, exit status 0
11:29:14 INFO  Executed sudo, exit status 0
11:29:19 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:29:19 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:29:19 INFO  Delaying execution by 8 seconds
11:29:41 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:29:45 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:29:45 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:29:45 INFO  
11:29:45 INFO  Test duration: 84.75 secs
11:29:45 INFO  Test final outcome:  BLOCKED
11:29:45 INFO  Test outcome reason: AssertionError:
11:29:45 INFO  
</system-out>
	</testcase>
</testsuite>
